### ISSUE TITLE
T keyboard shortcut no longer functions on new docs layout

### ISSUE THREAD

**Issue by ast-ral** [rust-lang/rust#94048]
In the "?" menu in the nightly* stdlib docs, it's documented that pressing "T" should "Focus the theme picker menu", however this menu appears to no longer exist.

\* specific version: `1.60.0-nightly (09cb29c64 2022-02-15)`

@rustbot label P-low T-rustdoc

**Comment by hellow554** [comment]
The "All crates" drop down has also disappeared:

Beta:
![grafik](https://user-images.githubusercontent.com/921462/154264365-dd1b7906-d7d0-4858-8559-7b25395c9d29.png)

Nightly:
![grafik](https://user-images.githubusercontent.com/921462/154264398-94df02d7-8bd7-4a3a-97a2-60bfda47bc8e.png)

@rustbot modify labels: regression-from-beta-to-nightly

**Comment by GuillaumeGomez** [comment]
The "All crates" drop down is now directly in the search result page (the PR is [here](https://github.com/rust-lang/rust/pull/92490)).

As for the theme picking menu, it's in the settings page now. Only exception: it is displayed when browsing docs with `file:///` URL (the PR is [here](https://github.com/rust-lang/rust/pull/92629)).

**Comment by ast-ral** [comment]
Right, I know about the theme options in the settings menu. This is just a minor gripe about the "?" menu's documentation not matching the real behavior of the "T" key. It says it should open up a theme picker menu (and it does still open the paintbrush in `file:///` URLs) but on the online docs it appears to actually do nothing (presumably because the paintbrush is hidden).

**Comment by GuillaumeGomez** [comment]
I'm waiting for https://github.com/rust-lang/rust/pull/93097 to be merged to remove it (it should be removed in the PR directly).
